src/components/onboarding/company.tsx

```diff
@@ -1,10 +1,84 @@
 "use client";
+
 import { Navbar } from "@/components/navbar";
 import { Button } from "@/components/ui/button";
 import { Label } from "@/components/ui/label";
 import { Input } from "@/components/ui/input";
+import * as z from "zod";
+import { zodResolver } from "@hookform/resolvers/zod"
+import { useForm } from "react-hook-form"
+
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form"
+
+const onboardingSchema = z.object({
+  firstName: z.string().min(1, {
```

Comment:
Lets design the schema in this way 👇🏼 , separating user and company
```
user: {
  firstName: z.string().min(1, {
    message: "First name is required",
  }),
  ...
},

company: {
  name: z.string().min(1, {
    message: "Company name is required",
  }),
  ...,
}
```

Also, lets design db schema as accordingly, change `org => company`

Reply:
👍